app/views/users/show.html.erb

```diff
@@ -0,0 +1,29 @@
+<section class="d-flex w-100 pt-5 pb-5 justify-content-center">
+  <div class="w-50">
+    <% user = User.where(id: params[:id])[0] %>
+    <% if user %>
```

Comment:
You can create field variable `@user` in the controller and if the user doesn't exist we would render 404 page or something of the sort.